lib/oban/telemetry.ex

```diff
@@ -139,6 +139,25 @@ defmodule Oban.Telemetry do
   | `:stop`      | `:duration`     | `:conf`, `:plugin`                                    |
   | `:exception` | `:duration`     | `:conf`, `:plugin`, `:kind`, `:reason`, `:stacktrace` |
 
+  ### Peer Events
+
+  Oban emits a telemetry span event each time an Oban Peer election occurs:
+
+  * `[:oban, :peer, :election, :start | :stop | :exception]`
+
+  | event        | measures       | metadata                                                       |
+  | ------------ | -------------- | -------------------------------------------------------        |
+  | `:start`     | `:system_time` | `:conf`, `:leader`, `:peer`,                        |
+  | `:stop`      | `:duration`    | `:conf`, `:leader`, `:peer`,                        |
+  | `:exception` | `:duration`    | `:conf`, `:peer`, `:kind`, `:reason`, `:stacktrace` |
```

Comment:
```suggestion
  | `:exception` | `:duration`    | `:conf`, `:leader`, `:peer`, `:kind`, `:reason`, `:stacktrace` |
```
It would be good to double-check this, but from what I'm reading in the [`:telemetry.span/3` docs](https://hexdocs.pm/telemetry/1.1.0/telemetry.html#span/3), the `:exception` event should include all of the metadata from the start event + the exception specific metadata.

Reply:
You're correct (and caught me between edits)!